docs/src/FAQ.md

```diff
@@ -76,3 +76,93 @@ ChainRulesTestUtils.jl has some dependencies, so it is a separate package from C
 This means your package can depend on the light-weight ChainRulesCore.jl, and make ChainRulesTestUtils.jl a test-only dependency.
 
 Remember to read the section on [On writing good `rrule` / `frule` methods](@ref).
+
+## How do chain rules work for complex functions?
+
+ChainRules follows the convention that `frule` applied to a function ``f(x + i y) = u(x,y) + i v(x,y)`` with perturbation ``\Delta x + i \Delta y`` returns the value and
+```math
+\tfrac{\partial u}{\partial x} \, \Delta x + \tfrac{\partial u}{\partial y} \, \Delta y + i \, \Bigl( \tfrac{\partial v}{\partial x} \, \Delta x + \tfrac{\partial v}{\partial y} \, \Delta y \Bigr)
+.
+```
+Similarly, `rrule` applied to the same function returns the value and a pullback function which, when applied to the adjoint ``\Delta u + i \Delta v``, returns
+```math
+\Delta u \, \tfrac{\partial u}{\partial x} + \Delta v \, \tfrac{\partial v}{\partial x} + i \, \Bigl(\Delta u \, \tfrac{\partial u }{\partial y} + \Delta v \, \tfrac{\partial v}{\partial y} \Bigr)
+.
+```
+If we interpret complex numbers as vectors in ``\mathbb{R}^2``, then `frule` (`rrule`) corresponds to multiplication with the (transposed) Jacobian of ``f(z)``, i.e. `frule` corresponds to
+```math
+\begin{pmatrix}
+\tfrac{\partial u}{\partial x} \, \Delta x + \tfrac{\partial u}{\partial y} \, \Delta y
+\\
+\tfrac{\partial v}{\partial x} \, \Delta x + \tfrac{\partial v}{\partial y} \, \Delta y
+\end{pmatrix}
+=
+\begin{pmatrix}
+\tfrac{\partial u}{\partial x} & \tfrac{\partial u}{\partial y} \\
+\tfrac{\partial v}{\partial x} & \tfrac{\partial v}{\partial y} \\
+\end{pmatrix}
+\begin{pmatrix}
+\Delta x \\ \Delta y
+\end{pmatrix}
+
+```
+and `rrule` corresponds to
+```math
+\begin{pmatrix}
+\tfrac{\partial u}{\partial x} \, \Delta u + \tfrac{\partial v}{\partial x} \, \Delta v
+\\
+\tfrac{\partial u}{\partial y} \, \Delta u + \tfrac{\partial v}{\partial y} \, \Delta v
+\end{pmatrix}
+=
+\begin{pmatrix}
+\tfrac{\partial u}{\partial x} & \tfrac{\partial v}{\partial x} \\
+\tfrac{\partial u}{\partial y} & \tfrac{\partial v}{\partial y} \\
+\end{pmatrix}
+\begin{pmatrix}
+\Delta u \\ \Delta v
+\end{pmatrix}
```

Comment:
> Doesn't this address the question about the conjugate? It's only the scalar case, but I guess it is clear that if we conjugate scalars than we also conjugate matrices.

Sorry, I had looked, but I missed that line. I think in that case, the Jacobians shown aren't right.

I think the above lines should actually be
```suggestion
\begin{pmatrix}
conj\left(\tfrac{\partial u}{\partial x}\right) \, \Delta u + conj\left(\tfrac{\partial v}{\partial x}\right)  \, \Delta v
\\
conj\left(\tfrac{\partial u}{\partial y}\right)  \, \Delta u + conj\left(\tfrac{\partial v}{\partial y}\right)  \, \Delta v
\end{pmatrix}
=
\begin{pmatrix}
conj\left(\tfrac{\partial u}{\partial x}\right)  & conj\left(\tfrac{\partial v}{\partial x}\right)  \\
conj\left(\tfrac{\partial u}{\partial y}\right)  & conj\left(\tfrac{\partial v}{\partial y}\right)  \\
\end{pmatrix}
\begin{pmatrix}
\Delta u \\ \Delta v
\end{pmatrix}
```